test/test_utils/__init__.py

```diff
@@ -71,7 +71,7 @@ def get_ami_id_ssm(region_name, parameter_path):
 NEURON_UBUNTU_18_BASE_DLAMI_US_WEST_2 = get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="Deep Learning Base AMI (Ubuntu 18.04) Version ??.?")
 UL20_PT_NEURON_US_WEST_2 = get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="Deep Learning AMI Neuron PyTorch 1.11.0 (Ubuntu 20.04) ????????")
 # Since NEURON TRN1 DLAMI is not released yet use a custom AMI
-NEURON_TRN1_AMI_US_WEST_2 = "ami-0a610a15fcc9e0242"
+NEURON_TRN1_AMI_US_WEST_2 = "ami-06a5a60d3801a57b7"
```

Comment:
can we use ami pattern to fetch the latest ami instead of hardcoding it? eg
`get_ami_id_boto3(region_name="us-west-2", ami_name_pattern="<>")`

Reply:
Done.